src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```
@@ -574,12 +579,44 @@ This package is an internal implementation of the .NET Core SDK and is not meant
   <Target Name="_BatchCopyToRedistLayout"
           Inputs="@(SharedFxContent)"
           Outputs="@(SharedFxContent->'$(RedistLayoutTargetDir)%(FileName)%(Extension)')">
+
     <Copy SourceFiles="@(SharedFxContent)"
           DestinationFiles="@(SharedFxContent->'$(RedistLayoutTargetDir)%(FileName)%(Extension)')"
           UseHardlinksIfPossible="true" />
     <Message Importance="High" Text="$(MSbuildProjectFile) -> $(RedistLayoutTargetDir)" />
   </Target>
 
+  <Target Name="_BatchCopyToCompositeLayout">
+    <PropertyGroup>
+      <CompositeFrameworkLayoutRoot>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', CompositeFx.Layout, '$(Configuration)', '$(TargetRuntimeIdentifier)'))</CompositeFrameworkLayoutRoot>
+      <LayoutAspnetPath>$(CompositeFrameworkLayoutRoot)\shared\$(SharedFxName)\$(SharedFxVersion)\</LayoutAspnetPath>
+      <LayoutNetcorePath>$(CompositeFrameworkLayoutRoot)\shared\Microsoft.NETCore.App\$(MicrosoftNETCoreAppRuntimeVersion)\</LayoutNetcorePath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <RedistLayoutFiles Include="$(RedistSharedFrameworkLayoutRoot)**\*.*" />
+      <CompositeFiles Include="$(CompositeTargetDir)**\*.*" />
+    </ItemGroup>
+
+    <RemoveDir Directories="$(CompositeFrameworkLayoutRoot)" />
+    <MakeDir Directories="$(CompositeFrameworkLayoutRoot)" />
+    <Copy SourceFiles="@(RedistLayoutFiles)" DestinationFolder="$(CompositeFrameworkLayoutRoot)\%(RecursiveDir)" />
```

Not a really big deal but the `%(DestinationFolder)` metadata ends up w/ two slashes at the end when `%(RecursiveDir)` is empty i.e. the files are placed directly in `$(CompositeFrameworkLayoutRoot)`. Suggest
```suggestion
    <Copy SourceFiles="@(RedistLayoutFiles)" DestinationFolder="$(CompositeFrameworkLayoutRoot)%(RecursiveDir)" />
```